violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/behavior/AddNodeBehavior.java

```diff
@@ -17,33 +15,62 @@
 import com.horstmann.violet.workspace.sidebar.graphtools.IGraphToolsBar;
 import com.horstmann.violet.workspace.sidebar.graphtools.IGraphToolsBarMouseListener;
 
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseEvent;
+import java.awt.geom.Point2D;
+import java.util.Collection;
+
+import static javax.swing.JOptionPane.showMessageDialog;
+
 public class AddNodeBehavior extends AbstractEditorPartBehavior implements IGraphToolsBarMouseListener
 {
-	private static final int OUTSIDE_SCREEN_POSITION = -1000;
+    /** default number of max initial nodes on workspace */
+    private static final int maxNumberOfInitialNodes = 1;
+    /** default screen position*/
+    private static final int OUTSIDE_SCREEN_POSITION = -1000;
+
+    private IEditorPart editorPart;
+
```

Comment:
why so many enters? (line 33-46) 
in this project, I think, the variables are at the end of a file - not here